subworkflows/local/long_reads_qc.nf

```diff
@@ -0,0 +1,95 @@
+include { FASTP as FASTP_LR                       } from '../../modules/nf-core/fastp/main'
+include { RAW_READ_QUALITY_CHECK                  } from '../../modules/local/raw_read_quality_check/'
+include { MINIMAP2_ALIGN as HUMAN_DECONTAMINATION } from '../../modules/nf-core/minimap2/align/main'
+include { MINIMAP2_ALIGN as HOST_DECONTAMINATION  } from '../../modules/nf-core/minimap2/align/main'
+
+workflow LONG_READS_QC {
+
+    take:
+    reads                   // [ val(meta), path(reads) ]
+    reference_genome   // [ val(meta2), path(reference_genome) ]
+
+    main:
+    ch_versions = Channel.empty()
+
+    FASTP_LR(
+        reads,
+        [],
+        false,
+        false,
+        false,
+        false
+    )
+
+    ch_versions = ch_versions.mix(FASTP_LR.out.versions)
+
+    RAW_READ_QUALITY_CHECK(
+        FASTP_LR.out.json
+    )
+
+    decontaminated_reads = channel.empty()
+
+    if ( params.remove_human ) {
+        // TODO: make this consistent with short_reads
+        // can we use the same flag, even if one has phix but not the other?
+        // Check file extensions too
+
+        human_reference = Channel.fromPath( "${params.reference_genomes_folder}/${params.human_fasta_prefix}.fna", checkIfExists: true)
+            .collect().map {
+                files -> [ ["id": params.human_fasta_prefix], files ]
+            }
+
+        // TODO: can we change the way human/host are given via prefixes?
+
+        HUMAN_DECONTAMINATION(
+            FASTP_LR.out.reads,
+            human_reference,
+            "human",
+            true,
```

Comment:
Can you add comments on the parameters this is recieving?

For example:
```
HUMAN_DECONTAMINATION(
            FASTP_LR.out.reads,
            human_reference,
            "human",
            true, # Filter out
            "bai", # Output format...
            false,
            true
        )
```

Stuff like that, it makes reading the code much easier when multiples parameters are send to a module like this